rts/Map/SMF/SMFReadMap.cpp

```diff
@@ -61,6 +61,7 @@ CSMFReadMap::CSMFReadMap(const std::string& mapName): CEventClient("[CSMFReadMap
 	haveSplatDetailDistribTexture = (!mapInfo->smf.splatDetailTexName.empty() && !mapInfo->smf.splatDistrTexName.empty());
 	haveSplatNormalDistribTexture = false;
 
+	// FIXME: undefined behavior, destination object type 'MapTexture' is not TriviallyCopyable [bugprone-undefined-memory-manipulation]
```

Comment:
technically UB then, but uninteresting because this LOC runs before the textures are assigned.

since MapTexture zero-inits itself anyway the memset is actually redundant.

Reply:
No bugs found, but oh well :)  At least it's not a false positive